inc/cappuccino/utlru_cache.hpp

```diff
@@ -148,6 +148,26 @@ class utlru_cache
         return deleted_elements;
     }
 
+    /**
+     * Removes all elements from the cache (which are destroyed), leaving the container size 0.
+     * @return none
+     */
+    auto clear() -> void
+    {
+        if (m_used_size > 0)
+        {
+            std::lock_guard guard{m_lock};
+            std::iota(m_lru_list.begin(), m_lru_list.end(), 0);
+            m_lru_end = m_lru_list.begin();
+            m_elements.clear();
+            m_keyed_elements.clear();
+            m_lru_list.clear();
```

Comment:
This is going to break the cache entirely, they are pre-allocated data structures, try adding to your test inserting items after clearing the cache, it should segfault pretty hard